src/PhpSpec/Console/Application.php

```diff
@@ -147,6 +147,13 @@ protected function loadConfigurationFile(InputInterface $input, ServiceContainer
 
                     $extension->load($container);
                 }
+            } elseif ('matchers' === $key && is_array($val)) {
+                foreach ($val as $class) {
+                    $id = strtolower(str_replace('Matcher', '', substr($class, strrpos($class, '\\') + 1)));
```

Comment:
I'm not sure if removing namespace completely is a good idea.
When you use only matchers defined in your project - it would be OK. 

But if you use third-party matchers - class names (without namespaces) can be equal, and if you use two matchers with the same name the first will be overwritten. That can lead to issues that are hard to understand.